lib/Client.js

```diff
@@ -1,7 +1,8 @@
 "use strict";
-var Client = module.exports = function(options) {
+var Client = module.exports = function(options, auth) {
   options = options || { };
```

Comment:
I know this wasn't changed by your PR, but while we're here... is options misnamed since from the examples it's a string path? And if so, should `options = options || { };` be `path = path || '';`